auth/http.go

```diff
@@ -1405,6 +1430,51 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 	}
 }
 
+func httpHandlerExportUserGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+
+	users, err := central.Central.GetAuthenticatorIdentities(authaus.GetIdentitiesFlagNone)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	ident2perm, err := central.Central.GetPermits()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	groups, err := central.Central.GetRoleGroupDB().GetGroupsRaw()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+	}
+
+	groupsResp, err := getPermitsJSON(central, users, ident2perm)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+	}
+
+	var exportGroups []exportGroup
+	groupNametoID := make(map[string]int)
+	for _, group := range groups {
+		exportGroups = append(exportGroups, exportGroup{Id: int(group.ID), Name: group.Name, Permlist: group.PermList})
+		groupNametoID[group.Name] = int(group.ID)
+	}
+
+	var exportGroupUsers []exportGroupUser
+
+	for _, user := range groupsResp {
+		groups := make([]int, len(user.Groups))
+		for i, group := range user.Groups {
+			groups[i] = groupNametoID[group]
+		}
+		exportGroupUsers = append(exportGroupUsers, exportGroupUser{Id: user.Email, Groups: groups})
```

Comment:
I think we decided to put the field that it used to match the user on in front of the variable. If you think thats unnecessary then its probably fine. But it still feels like we should then specify the field name somewhere.